### PR TITLE
Adding Cross function to tensorflow raw_ops, addresses issue #3700

### PR DESCRIPTION
**Pull request by carlacampas**

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 1]
```diff
@@ -196,6 +196,34 @@ def test_tensorflow_Cosh(
...
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="tensorflow",
+        fn_tree="raw_ops.Cross",
+        x=np.asarray(x, dtype=input_dtype),
```
Comment (rush2406) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:224]: Cross takes 2 arguments so here also there should be 2 inputs, you can refer [here](https://github.com/unifyai/ivy/blob/45d5abc45ecfe815edc4664e20bafb2b2b050c47/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py#L569). I believe this should be able to solve the failing test.

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @carlacampas 
Thanks for contributing! There are a few small changes required please check the comments below. Also please fix the formatting issues causing lint failures and request a review once done.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py (hunk 1 above)
Comment (carlacampas) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py]: Thank you! Fixed :)

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 2]
```diff
@@ -197,6 +197,38 @@ def test_tensorflow_Cosh(
...
 
+# Cross
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
+        ),
+        num_arrays=2,
+        shared_dtype=True,
```
Comment (rush2406) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:208]: so it seems like the examples being generated for Cross are not correct. It requires 2 or 3-dimensional inputs. Can you add `min_num_dims`=2 and `max_num_dims`=3?

**Review by rush2406 — CHANGES_REQUESTED** [review]

**Comment by rush2406** [comment]
Also please fix the merge conflicts!

**Pull request closed without merging**